package.json

```diff
@@ -47,7 +47,7 @@
     "release": "npm run build && changeset publish",
     "start": "npm run build && npm run start:storybook & npm run start:docs",
     "start:storybook": "start-storybook -p 6006",
-    "start:docs": "cd docs && npm run develop",
+    "start:docs": "npm run build && cd docs && npm run develop",
```

Comment:
@JoshBowdenConcepts this will fix the issue that you encountered earlier today

Reply:
looks good to me!